enterprise/backend/src/metabase_enterprise/serialization/v2/storage.clj

```diff
@@ -1,15 +1,45 @@
 (ns metabase-enterprise.serialization.v2.storage
```

Comment:
This now contains the YAML storage implementation. If/when we decide to add some other kind of destination, we can cross that bridge when we come to it. It could probably just be a multimethod in this file, depending on how complicated it is.